### ISSUE TITLE
[MU4 Issue] MU doesn't see VST on Linux Mint (u-he Zebra2)

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#11550]
**Describe the bug**
A short and clear description of what the bug is.

**To Reproduce**
1. Install Linux Mint 20.2 Xfce Edition (from linuxmint.com)
2. Install VSTi from https://u-he.com/products/zebralette/
3. Open MU (use master build from GitHub and not local build)
4. Run MU 
5. Open a score and mixer

Old build shows VST available:
![Screenshot 2022-05-06 20_28_30](https://user-images.githubusercontent.com/90187801/167157914-417ba995-66a5-4b74-8057-20780b7d75b7.png)

New master build:

![Screenshot 2022-05-06 20_30_33](https://user-images.githubusercontent.com/90187801/167157970-31e7b675-4540-44d0-bebb-513640133274.png)

**Expected behavior**

**Screenshots**

**Platform information**
Linux Mint 20.2

**Additional context**
Can be related to #10836

**Comment by diedeno** [comment]
What's the point of this? 
VST GUi does not work anyway in Linux.
Perhaps it would be best to remove VST altogether from Linux appimage.

**Comment by DmitryArefiev** [comment]
> What's the point of this? VST GUi does not work anyway in Linux. Perhaps it would be best to remove VST altogether from Linux appimage.

@diedeno Yeah, agree with you. The VST feature in Linux builds will be blocked in #11689
